Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM python:3.9
+
+EXPOSE 9000
+WORKDIR /opt/storage-testbench
+
+COPY . /opt/storage-testbench/
+
+RUN python3 -m pip install -r requirements.txt
+
+CMD ["gunicorn", \
+      "--bind", "localhost:9000", \
+      "--worker-class", "sync", \
+      "--threads", "10", \
```

Comment:
Feels that it should be `$(nproc)` or something similar instead of the hard-coded `10`.

Reply:
CMD does a direct exec, it doesn't run it through a shell so we don't get shell expansion for free.

I can rewrite this cmd as a `/bin/bash -c '...'` or move this into a wrapper script and then call that script. Do you have a preference?

Reply:
If there is no easy way to do it, just commit and we can improve this later... There is no reason why "10" is ideal, but it is not a terrible choice either 🤷